jazz-ui/src/environments/environment.oss.ts

```diff
@@ -9,6 +9,7 @@ export const environment = {
   environmentTabs:["overview"],
   urls:{
     docs_link:"https://github.com/tmobile/jazz/wiki",
+    content_base: "https://raw.githubusercontent/tmobile/jazz/jazz-content/jazz-ui/user-journey/"
```

Comment:
The path seems to be of the format https://github.com/tmobile/jazz-content/blob/master/jazz-ui/user-journey/jazz-bitbucket-environment.mp4?raw=true